### ISSUE TITLE
Keep open to spread love to Mariusz for sharing this

### ISSUE THREAD

**Issue by mikemix** [mariuszgil/awesome-eventstorming#3]
Great job as always Mariusz, thanks for sharing!

**Comment by mariuszgil** [comment]
Thanks @mikemix! I'm excited to spread this idea and learn new things :)

**Comment by akondas** [comment]
Yep, thanks to @mariuszgil and @wjptak I can evangelize rest of the world. Thanks once again!
Also big thanks to @ziobrando for Event Storming :fire:.
